dll/win32/shell32/wine/pidl.c

```diff
@@ -555,8 +555,11 @@ BOOL WINAPI ILIsEqual(LPCITEMIDLIST pidl1, LPCITEMIDLIST pidl2)
     while (pidltemp1->mkid.cb && pidltemp2->mkid.cb)
     {
         if (!_ILHACKCompareSimpleIds(pidltemp1, pidltemp2))
+#ifndef __REACTOS__
             return FALSE;
-
+#else
+            return TRUE;
+#endif
```

Comment:
That part looks like a hack. What is the symptom without that change?

Reply:
ILIsEqual must return TRUE when objects are equal
If IsEqual returns FALSE the rename function is aborted.

Reply:
But I do assume that this code is not only run when renaming the Network Connections object.